src/components/cards/SquareCard/index.module.scss

```diff
@@ -200,4 +201,106 @@
             }
         }
     }
+}
+
+.pricing {
+    .bg {
```

Comment:
With css modules this class structure seems off, there's no need for the `.pricing` scope around all of these classes since they're auto-prefixed.

Reply:
I didn't want to mess with the css for the existing `squareCard` since it is used for the `cardGrid` block - so I added an appearance prop and put the pricing specific css under `.pricing` but maybe I should have made another card `PricingCard`?

Reply:
Yea either make a new card you make your props for generic, like pass `price` through as `leader` for instance (not a real prop, just using as an example). Otherwise we lose component isolation.

Reply:
Added a new card!